application/client/src/app/ui/layout/module.ts

```diff
@@ -12,6 +12,7 @@ import { MatDividerModule } from '@angular/material/divider';
 import { MatSnackBarModule } from '@angular/material/snack-bar';
 import { MatIconModule } from '@angular/material/icon';
 import { MatMenuModule } from '@angular/material/menu';
```

Comment:
We never used angular tooltips before. To keep UI/UX the same for the whole solution, I would exclude this change from this PR. If we would need to switch to angular tooltips, we should do this for everywhere tooltips are used.

Reply:
Removed the tooltip and altered the message.